### ISSUE TITLE
[1.12] Some ores don't exist, or used to exist and after an update, were removed?

### ISSUE THREAD

**Issue by Spearkiller** [nfinit-gaming/Nether-Metals#62]
In the config files it mentions things like Nether Coal ore, Nether Iridium, Nether Chromium, etc, yet they don't appear in game. I've looked over the config files in both this and base metals to make sure that the right things are enabled (and they are) yet these ores don't appear in game. Even after the most recent update all instances of Nether Coal, Emerald, (and their End variants), and potentially a few others, it's hard to tell due to them not being in the mod anymore, just vanished from my world and they aren't in JEI or the Creative Inventory.
I do have things like Mercury, Antimony and all that disabled but other ores are not.
The textures even exist in the files so I'm just not getting why they don't exist in game?

Base Metals Version: 1.12-2.5.0-beta6.280
Nether Metals Version: 1.12-1.2.0-beta4.65
End Metals Version: 1.12-1.2.0-beta4.53

**Comment by YaibaToKen** [comment]
Aren't those ores introduced by Modern Metals?

**Comment by jriwanek** [comment]
Coal isn't, I'll double check the behaviours however

**Comment by YaibaToKen** [comment]
> Coal isn't, I'll double check the behaviours however

you're right about coal though I do have it being generated in my DS with the latest (stable) version of the mod

**Comment by Spearkiller** [comment]
Figured out both of the issues; If you disable, for example, Emerald Items and Materials in the Base Metals config, it disabled the relevant End Metals and Nether Metals. Same goes for coal and anything else.

As for the ores, they are in modern metals, I just got confused because there was config for things like Nether Osmium ore generation, and, even though I had a mod that added Osmium (Mekanism), the nether and end ore still wasnt generating. Maybe there could be a way for the mod to check if you have ores like Osmium/Titanium/etc. in the mods and allow those ores to spawn, but not others, making it not so reliant on Base/Modern Metals, but still supporting it?
